sklearn/model_selection/_validation.py

```diff
@@ -489,6 +511,8 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
         ret.extend([fit_time, score_time])
     if return_parameters:
         ret.append(parameters)
+    if return_estimator:
+        ret.append(estimator)
```

Comment:
Apparently coverage is missing from this line!

Reply:
I have added an `assert False` along with my `assert_almost_equal` on the estimator parameters and this makes the test fail, so they are run. So I don't understand how the above line cannot be covered

Reply:
perhaps put the assert False here just to prove the coverage tool wrong? If it does not fail, you've got some investigation to do...

Reply:
Thanks, just did this and the test now fails. Should I commit this for the sake of it?

Reply:
How strange...

Reply:
How strange...